src/internal/connector/sharepoint/data_collections.go

```diff
@@ -71,17 +81,58 @@ func DataCollections(
 				if err != nil {
 					return nil, support.WrapAndAppend(site, err, errs)
 				}
-
-				collections = append(collections, spcs...)
 			}
 
+			collections = append(collections, spcs...)
+
 			foldersComplete <- struct{}{}
 		}
 	}
 
 	return collections, errs
 }
 
+func collectLists(
+	ctx context.Context,
+	serv graph.Servicer,
+	tenantID, siteID string,
+	scope selectors.SharePointScope,
+	updater statusUpdater,
+	ctrlOpts control.Options,
+) ([]data.Collection, error) {
+	logger.Ctx(ctx).With("site", siteID).Debug("Creating SharePoint List Collections")
+
+	if scope.Matches(selectors.SharePointSite, siteID) {
+		spcs := make([]data.Collection, 0)
+
+		listIDs, err := preFetchListIDs(ctx, serv, siteID)
+		if err != nil {
+			return nil, err
+		}
+		// Prefetch
+		for _, listID := range listIDs {
+			dir, err := path.Builder{}.Append(listID).
```

Comment:
do lists have meaningful names associated with them that we need to preserve? We don't _have_ to store it as the folder name if they do, but we should have a plan for how we do plan to store it if we need to

Reply:
It is unclear whether the `displayName` would be of use to the user. These are the list we are returning at present. 
- Access Requests
- Events
- SharePointHomeOrgLinks
- Documents

Reply:
Resolution within: [ae26725](https://github.com/alcionai/corso/pull/1788/commits/ae267258e36aa5c10c2fdd2280d6905e04e87154)